qml/PingSlider.qml

```diff
@@ -4,15 +4,14 @@ import QtQuick.Controls.Styles 1.0
 import QtQuick.Layouts 1.3
 import QtQuick.Controls.Material 2.1
 
-import SliderRuler 1.0
-
 RowLayout {
     id: root
 
     property var text
     property double value: 0
     property double from: 0
     property double to: 0
+    property double ticksNumber: 6
```

Comment:
what about just making it 7 instead of using `ticksNumber + 1` in the model? We do get 7 ticks after all.

Reply:
This property allow us to change the ticks number, the model will always be "ticks number" + 1;